README.md

```diff
@@ -33,12 +33,12 @@ To install this gem onto your local machine, run `bundle exec rake install`. To
 
 ## Contributing
 
-Bug reports and pull requests are welcome on GitHub at https://github.com/controlshift/mobilizeamerica-client. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct.
+Bug reports and pull requests are welcome on GitHub at https://github.com/controlshift/mobilize-america-client. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct.
 
 ## License
 
 The gem is available as open source under the terms of the [MIT License](https://opensource.org/licenses/MIT).
 
 ## Code of Conduct
 
-Everyone interacting in the Mobilizeamerica::Client project’s codebases, issue trackers, chat rooms and mailing lists is expected to follow the [code of conduct](https://github.com/controlshift/mobilizeamerica-client/blob/master/CODE_OF_CONDUCT.md).
+Everyone interacting in the MobilizeAmericaClient::Client project’s codebases, issue trackers, chat rooms and mailing lists is expected to follow the [code of conduct](https://github.com/controlshift/mobilize-america-client/blob/master/CODE_OF_CONDUCT.md).
```

Comment:
Do we need the namespacing here, or can it just be "in the MobilizeAmericaClient project's"?